test_runner/src/test/kotlin/TmpV2.kt

```diff
@@ -0,0 +1,64 @@
+import com.google.api.client.json.GenericJson
+import ftl.args.AndroidArgs
+import ftl.gc.GcTestMatrix
+import ftl.reports.api.createJUnitTestResult
+import ftl.reports.api.createTestExecutionDataListAsync
+import ftl.reports.api.data.TestExecutionData
+import ftl.reports.api.filterForJUnitResult
+import ftl.reports.xml.xmlToString
+import java.io.File
+import kotlin.system.exitProcess
+
+object TmpV2 {
```

Comment:
we should probably delete `TmpV2` and turn this into a proper JUnit test. What do you think?

Reply:
Also probably delete the original tmp

Reply:
Of Course I will do that. This is only draft of implementation that I tested manually and looks correct. I now need to update unit tests, cleanup a little, and It will be ready for review.